app/assets/javascripts/application.js

```diff
@@ -7,7 +7,6 @@
 //= require patternfly
 //= require vendor
 //= require jquery.extentions
-//= require jquery.multi-select
```

Comment:
the actual file (app/assets/javascripts/jquery.multi-select.js) should also be removed.